manifests/e/ente-io/auth-desktop/3.0.4/ente-io.auth-desktop.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ente-io.auth-desktop
+PackageVersion: 3.0.4
+
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Scope: machine
+InstallerSwitches: 
+  Silent: /VERYSILENT /SUPPRESSMSGBOXES
+  SilentWithProgress: /SILENT /SUPPRESSMSGBOXES
+ElevationRequirement: elevatesSelf
+UpgradeBehavior: install
+ReleaseDate: 2024-05-21
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/ente-io/ente/releases/download/auth-v3.0.4/ente-auth-v3.0.4-installer.exe
+  InstallerSha256: 8BC0CA6CE6291EFEA576B458076EECD8BC07C7730C5FEA7B743FEE4370D6453C
+

```

Comment:
```suggestion
  InstallerSha256: 8BC0CA6CE6291EFEA576B458076EECD8BC07C7730C5FEA7B743FEE4370D6453C
  AppsAndFeaturesEntries:
    - DisplayVersion: '3.0.4+304'
```

Reply:
What does this do?

Reply:
This tells the package manager to display `3.0.4` in output responses and match the packaging amd marketing version, while matching with what the Registry shows as `3.0.4+304`. It's useful, for example, when the application's About or --version shows one number, but the Apps and features Settings page shows another number.